custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py

```diff
@@ -51,7 +105,15 @@ def drop_table_query(self):
             DELETE FROM "{tablename}";
         """.format(tablename=self.base_tablename)
 
-    def aggregate_query(self):
+    def aggregate_to_temporary_table(self, cursor, csv_file):
+        cursor.copy_from(csv_file, 'temp_awc_location')
+
+    def create_temporary_table_query(self):
+        return "CREATE TEMPORARY TABLE \"temp_awc_location\" AS SELECT * FROM \"{tablename}\"".format(
```

Comment:
why does this get populated from the original table?

Reply:
that was a mistake in my initial implementation. this was fixed in monoltih query, but i hadn't fixed up the distributed one